src/components/cubes.jsx

```diff
@@ -29,34 +36,58 @@ export function Cubes({ app, closeOnClickOutside }) {
         newCube,
         ...cubeList,
       ]);
+      refs.current[newCube.id] = React.createRef();
     }
     addOpen.current = false;
     forceUpdate();
   }
 
   function removeCube(id) {
     setCubeList(cubeList.filter(item => item.id !== id));
+    refs.current[id] = null;
   }
 
   function openColumnChooser() {
     addOpen.current = true;
     forceUpdate();
   }
 
+  function copyToClipboard(id) {
+    if (refs.current && refs.current[id] && refs.current[id].current) {
```

Comment:
Does our babel implementation allow optional chaining? https://github.com/tc39/proposal-optional-chaining

E.g. `refs.current?[id]?.current`

Reply:
Sadly not, seems to be in the proposal phase still..